app/src/main/java/asd/org/ahorcado/dal/DataBaseHelper.java

```diff
@@ -0,0 +1,115 @@
+package asd.org.ahorcado.dal;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteException;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.sql.SQLException;
+
+import asd.org.ahorcado.BuildConfig;
+
+/**
+ * Created by Huber on 17/04/2016.
+ */
+public class DataBaseHelper extends SQLiteOpenHelper {
+    private Context mycontext;
+    private static String DB_NAME = "Ahorcado.db";
+    private static String DB_PATH ="/data/data/"+ BuildConfig.APPLICATION_ID+"/databases/";
+    public SQLiteDatabase myDataBase;
+
+    public DataBaseHelper(Context context) throws IOException {
+        super(context,DB_NAME,null,1);
+        this.mycontext=context;
+        boolean dbexist = checkdatabase();
+        if (dbexist) {
+            System.out.println("Database exists");
+            try {
+                opendatabase();
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        } else {
+            System.out.println("Database doesn't exist");
+            createdatabase();
+        }
+    }
+
+    public void createdatabase() throws IOException {
+        boolean dbexist = checkdatabase();
+        if(dbexist) {
+            System.out.println(" Database exists.");
+        } else {
+            this.getReadableDatabase();
+            try {
+                copydatabase();
+            } catch(IOException e) {
+                throw new Error("Error copying database");
+            }
+        }
+    }
+
+    private boolean checkdatabase() {
+
```

Comment:
No no, you did not unsterstand me :P. A file helper is useful to isolate behaviours related to files in other object and use them in differents parts